features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml.j2

```diff
@@ -49,6 +49,14 @@
                    description="Delete API resources"/>
         </Scopes>
     </APIResource>
+    <APIResource name="API Resource Management API" identifier="/o/api/server/v1/api-resources"
```

Comment:
shall we update the xml files as well for consistency

Reply:
Added